packages/frama-c/frama-c.20180501/opam

```diff
@@ -93,6 +93,10 @@ build-doc: [
    [make "-C" "doc" "install"]
 ]
 
+build-test: [
```

Comment:
This patch cannot be merged into `opam-repository` until the next release of Frama-C, because opam's CI runs these tests and they will fail due to differences in the open source version. The next release (Frama-C 18 - Argon) will incorporate it.